docs: rebrand Vertex AI to Gemini Enterprise Agent Platform in AI logic Skills (#157)
firebase/agent-skills: skills/firebase-ai-logic-basics/references/flutter_setup.md

> [!IMPORTANT] **Choose the Right API Provider:** Always use
> `FirebaseAI.googleAI` (Gemini Developer API) as the default for prototyping
> and standard use. Avoid using the Agent Platform Gemini API (formerly branded
> Vertex AI) unless the app and business use case specifically requires
> enterprise-grade scalability or data residency requirements. Note that the
> Agent Platform Gemini API *requires* the Firebase project to be on the

> > pay-as-you-go Blaze pricing plan.

Reviewer: ```suggestion
> and standard use. Avoid using the Agent Platform Gemini API (formerly branded
> Vertex AI) unless the app and business use case specifically require
> enterprise-grade scalability or data residency requirements. Note that the
> Gemini Developer API *usually does not* require the Firebase project to be on
> the pay-as-you-go Blaze pricing plan; however, the Agent Platform Gemini API
> does require the Blaze plan.
```

> [!IMPORTANT] **Choose the Right API Provider:** Always use
> `FirebaseAI.googleAI` (Gemini Developer API) as the default for prototyping
> and standard use. Avoid using the Agent Platform Gemini API (formerly branded

> > Vertex AI) unless your application specifically requires enterprise-grade

Reviewer: ```suggestion
> Vertex AI) unless the app and business use case specifically requires enterprise-grade
```

Reviewer: Agent Skills are for the *agent*, not the human.  So we need to use phrasing where we're talking to the agent :-)

> > scalability and is configured for the Blaze plan.

Reviewer: ```suggestion
> scalability or data residency requirements. Note that the Agent Platform Gemini API *requires* the Firebase project to be on the pay-as-you-go Blaze pricing plan.
```
